src/common/logger/logger.util.ts

```
@@ -0,0 +1,79 @@
+import * as winston from 'winston';
+
+export class LoggerWinston {
+  private static logger: winston.Logger;
+
+  static getLogger() {
+    if (!this.logger) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)